target/linux/realtek/files-5.10/include/dt-bindings/clock/rtl83xx-clk.h

```diff
@@ -0,0 +1,14 @@
+/* SPDX-License-Identifier: GPL-2.0-only */
+/*
+ * Copyright (C) 2022 Markus Stockhausen
+ *
+ * RTL83XX clock indices
+ */
+#ifndef __DT_BINDINGS_CLOCK_RTL83XX_H
+#define __DT_BINDINGS_CLOCK_RTL83XX_H
```

Comment:
```suggestion
#define __DT_BINDINGS_CLOCK_RTL83XX_H __FILE__
```
I've learned this trick a while ago and quite like it; it means it IS define, has ACTUAL content and nothing too random :)

Reply:
Linux kernel dows not use __FILE__ anywhere in the DT.